aiida/schedulers/plugins/lsf.py

```diff
@@ -599,9 +601,9 @@ def _parse_joblist_output(self, retval, stdout, stderr):
 
             this_job.queue_name = partition
 
-            psd_finish_time = self._parse_time_string(finish_time, fmt='%b %d %H:%M')
-            psd_start_time = self._parse_time_string(start_time, fmt='%b %d %H:%M')
-            psd_submission_time = self._parse_time_string(submission_time, fmt='%b %d %H:%M')
+            psd_finish_time = self._parse_time_string(finish_time, fmt='%b %d %H:%M:%S %Y')
+            psd_start_time = self._parse_time_string(start_time, fmt='%b %d %H:%M:%S %Y')
+            psd_submission_time = self._parse_time_string(submission_time, fmt='%b %d %H:%M:%S %Y')
```

Comment:
Why are you also adding the seconds in the format here? You are only expecting the year to be added no?